docfx/index.md

```diff
@@ -14,6 +14,8 @@ layout: landing
 | [IceRpc.Locator]                        | Provides the locator interceptor. This interceptor enables interop with Ice servers registered with a locator such as IceGrid.                     |
 | [IceRpc.Logger]                         | Provides the logger interceptor and middleware, for logging requests and responses to an ILogger.                                                  |
 | [IceRpc.Metrics]                        | Provides the metrics interceptor and middleware.                                                                                                   |
+| [IceRpc.Protobuf]                       | Provides support for the IceRPC + Protobuf integration.                                                                                            |
+| [IceRpc.Protobuf.Generators]            | Provides the Protobuf code generators for the IceRPC + Protobuf integration.                                                                       |
```

Comment:
Should we really include IceRpc.Protobuf.Generators and IceRpc.Slice.Generators in the public API reference?

I think it would be more confusing than helpful.

Reply:
I've removed this entry and excluded the both Slice and Protobuf generators.